oas_docs/output/kibana.yaml

```diff
@@ -84215,6 +84267,36 @@ components:
       required:
         - has_all_required
         - privileges
+    Security_Entity_Analytics_API_EntityField:
+      type: object
+      properties:
+        attributes:
+          type: object
+          properties:
+            privileged:
+              type: boolean
```

Comment:
What happens when we `force` an update on a field that is not mapped, for example `entity.attributes.asset.criticality`? will it be mapped dynamically?

Reply:
<img width="1708" height="283" alt="image" src="https://github.com/user-attachments/assets/b091aae4-160b-4b70-a8b2-affa378e4a06" />
I have made the request strict [here ](https://github.com/elastic/kibana/pull/234454/commits/fb7020037c714946f2320e8bc01b93adfeba5f55)

Reply:
The API documentation will have it!